ceph/ceph/values.yaml

```diff
@@ -276,12 +274,28 @@ bootstrap:
     }
     ensure_pool volumes 8
 
-ceph_local_pool:
-  failure_domain: host
-  subtree: rack
-  pg_num: "128"
-  num_rep: "3"
-  min_size: "2"
+# Uncomment below to enable mgr modules
+# For a list of available modules:
+#  http://docs.ceph.com/docs/master/mgr/
+# This overrides mgr_initial_modules. Any
+# module not listed here will be disabled
+ceph_mgr_enabled_modules:
```

Comment:
would not hurt to mention that this list is the same list enabled by default outside of Helm